### ISSUE TITLE
Not saving canvas

### ISSUE THREAD

**Issue by Danish00786** [AlecAivazis/feynman#109]
When i draw feyman diagram online and try to save then it gives error.
Unable to open document “file:///tmp/mozilla_danish0/An Example Diagram-2.pdf”.
How to fix it?

**Comment by David-Ellis** [comment]
Did you ever find a solution? I still can't seem to save it.

**Comment by Danish00786** [comment]
Yes I have found it. In my case error was due to some script issues. For example after written e^+ instead of e^{+} my issue resolved. So you must check  syntax of labels. Make some suitable changes to remove errors.

**Comment by David-Ellis** [comment]
Thank you! It turns out that despite the fact that the website displays the latex text correctly, you still need "$" on either side to enter math-mode.
